FirebaseInAppMessaging/Sources/RenderingObjects/FIRInAppMessagingRenderingPrivate.h

```diff
@@ -91,7 +68,7 @@ NS_ASSUME_NONNULL_BEGIN
               renderAsTestMessage:(BOOL)renderAsTestMessage
                       triggerType:(FIRInAppMessagingDisplayTriggerType)triggerType
                         titleText:(NSString *)title
-                         bodyText:(NSString *)bodyText
+                         bodyText:(nullable NSString *)bodyText
```

Comment:
Is this an expected change?

Reply:
Yeah, I noticed there was a nullability error there. All `bodyText` params in in-app messages should be nullable.